gdk4/src/toplevel_size.rs

```diff
@@ -0,0 +1,111 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+#[derive(Clone, Copy, Debug)]
+#[repr(C)]
+pub struct ToplevelSize(ffi::GdkToplevelSize);
+
+impl ToplevelSize {
```

Comment:
Should maybe also get a constructor and accessors for the other fields

Reply:
At least accessors for the fields would be useful then, no?

Reply:
I asked about this in iirc

> ebassi: It's an opaque structure
> ebassi: You're supposed to set it up using the given methods
> ebassi: And that's all